zerver/management/commands/compilemessages.py

```diff
@@ -144,4 +144,11 @@ def get_translation_percentage(self, locale_path: Text, locale: Text) -> int:
                 if value == '':
                     not_translated += 1
 
+        # mobile stats
+        with open(os.path.join(locale_path, 'mobile_info.json')) as mob:
+            mobile_info = ujson.load(mob)
+        info = mobile_info.get(locale, {'total': 0, 'not_translated': 0})
```

Comment:
I think it might be better to throw an error if the `mobile_info` doesn't have the locale.

Reply:
Done.

Reply:
CI fails if we we throw an error.